src/main/webapp/app/course/manage/image-cropper-modal.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component } from '@angular/core';
+import { ArtemisSharedCommonModule } from 'app/shared/shared-common.module';
+import { ImageCropperModule } from 'app/shared/image-cropper/image-cropper.module';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { ImageCroppedEvent } from 'app/shared/image-cropper/interfaces/image-cropped-event.interface';
+
+@Component({
+    selector: 'jhi-image-cropper-modal',
+    standalone: true,
```

Comment:
Since Patrick wanted to create it as a modal I thought it could be more straightforward to transfer the information like this, however since this modal is probably not going to be used anywhere else, I can include it in a module, yes

Reply:
I think that would make sense 👍 We don't have many standalone components in the project